option go_package = "a.yandex-team.ru/kernel/qtree/richrequest/markup/wtypes/proto;wptypespb";

enum EOntoCatsType {
    option allow_alias = true;
    ONTO_CATS_BEGIN = 0;

    ONTO_UNKNOWN    = 0;
    ONTO_ORG        = 1;
    ONTO_GEO        = 2;
    ONTO_HUM        = 3;
    ONTO_SOFT       = 4;
    ONTO_TEXT       = 5;
    ONTO_DEVICE     = 6;
    ONTO_MUSIC      = 7;
    ONTO_SITE       = 8;
    ONTO_FILM       = 9;
    ONTO_BAND       = 10;
    ONTO_AUTO       = 11;
    ONTO_DATE       = 12;
    ONTO_DISEASE    = 13;
    ONTO_EVENT      = 14;
    ONTO_DRUGS      = 15;
    ONTO_FOOD       = 16;
    ONTO_PICTURE    = 17;
    ONTO_AUTO_PARTS = 18;
    ONTO_CLOTHES    = 19;
    ONTO_FURNITURE  = 20;
    ONTO_TOYS       = 21;
    ONTO_ANIM       = 22;
    ONTO_ACCOM      = 23;
    ONTO_TRANS      = 24;
    ONTO_INTENT     = 25;
    ONTO_VOTING_CONT = 26;
    ONTO_BEAUTY     = 27;

    ONTO_CATS_END   = 28;    // should be last
}

enum EOntoIntsType {
    option allow_alias = true;
    ONTO_INTS_BEGIN      = 0;

    ONTO_INTS_UNKNOWN    = 0;
    ONTO_INTS_AUTO        = 1;
    ONTO_INTS_AUTO_PARTS  = 2;
    ONTO_INTS_DEVICE      = 3;
    ONTO_INTS_LIVE        = 4;
    ONTO_INTS_MUSIC       = 5;
    ONTO_INTS_PLAY        = 6;
    ONTO_INTS_TEXT        = 7;
    ONTO_INTS_SPORT       = 8;
    ONTO_INTS_FILM        = 9;
    ONTO_INTS_IMPORTED    = 10;
    ONTO_INTS_JOB         = 11;
    ONTO_INTS_CLOTHES     = 12;

    ONTO_INTS_END         = 13;     // should be last
}

enum EOntoVotingContextType {
    ONTO_VC_UNKNOWN    = 0;
    ONTO_VC_UNDIRECTED = 1;
    ONTO_VC_TO_RIGHT   = 2;
    ONTO_VC_TO_LEFT    = 3;
}



